### DISCUSSION TITLE
Loss spikes and does not go down on lj speech

### DISCUSSION THREAD

**Discussion by JRMeyer** [coqui-ai/TTS#320] [Archived]
<USER>>>> Daniil_Balenko<br/> <DATE> [December 8, 2020, 8:51am]<br/> <br/> Hi,<br/> <br/> I am trying to train tacotron from scratch, using LJ Speech and the<br/> [config from current<br/> master.](https://github.com/mozilla/TTS/blob/1229554c427567f1b09318ddd3732ad463efebbe/TTS/tts/configs/config.json)<br/> <br/> Some time after scheduled `r` reduction from 7 to 5 after 50k steps,<br/> losses across the board spike and stay flat: *slash* <br/> <br/> *slash* <br/> <br/> I'm not a machine learning expert, and would appreciate any help.<br/> <br/> [This is an archived TTS discussion thread from discourse.mozilla.org/t/loss-spikes-and-does-not-go-down-on-lj-speech]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> Daniil_Balenko<br/> <DATE> [December 22, 2020, 9:15pm]<br/> <br/> Trying to bump the topic, as it did not get any attention.<br/> <br/> was there an attempt to build<br/> a model from scratch with new additions since [the last<br/> model](https://drive.google.com/drive/folders/1Y_0PcB7W6apQChXtbt6v3fAiNwVf4ER5)<br/> was released?<br/> <br/> Also, what set of hyperparameters should I use if I want to disable<br/> newly added behaviors, like spectral and ssim loss, but only use mixed<br/> precision training? I see that GA and stopnet loss weights were adjusted<br/> as well, and simply setting the former to 0 does not lead to model<br/> convergence.<br/> <br/> [Archived Post]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> erogol<br/> <DATE> [December 22, 2020, 11:41pm]<br/> <br/> I am not sure what are good values for your specific problem but I<br/> suggest you to start from <https://github.com/erogol/TTS_recipes> and<br/> edit one of them for your dataset. I'd give you a good start.<br/> <br/> [Archived Post]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> Daniil_Balenko<br/> <DATE> [December 23, 2020, 12:04am]<br/> <br/> The graphs that I've provided and the dataset that I'm using for<br/> training now is vanilla LJSpeech.<br/> <br/> My goal though is to fine-tune an LJSpeech model on my custom dataset.<br/> Mixed precision would help to get a batch size of 64 on my GTX 1070, but<br/> it was added only recently. Released model is trained on a much older<br/> revision, so I decided to re-train it on LJSpeech from scratch on the<br/> current master.<br/> <br/> I was wondering if there were any newer experiments with LJSpeech that<br/> yielded good results, and if current config reflects that.<br/> <br/> [Archived Post]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> erogol<br/> <DATE> [December 23, 2020, 10:54am]<br/> <br/> The best config is always with the latest model released. But depending<br/> on your technical level, I'd still suggest to start with the receipt.<br/> <br/> [Archived Post]<br/>
